org-static-blog.el

```diff
@@ -182,6 +182,16 @@ See also `org-static-blog-preview-ellipsis' and
   :type '(boolean)
```

Comment:
Should perhaps add a reference to the variables `org-static-blog-preview-start` and `org-static-blog-preview-end`, to clarify the behavior with and without them.